website/content/en/docs/concepts/provisioner.md

```diff
@@ -0,0 +1,6 @@
+---
+title: "Karpenter provisioner and scheduler"
```

Comment:
What's the goal of this section?

Reply:
I was thinking of stepping through a couple of provisioner CRDs that are applied to the same cluster. It could illustrate different teams on the same cluster having whole separate sets of requirements. Then I could show examples of particular pod specs in the "Supported Scheduling Constraints" section and what the results might be from those provisioners.

Reply:
Your thoughts?